### ISSUE TITLE
Seek for Trained-Models

### ISSUE THREAD

**Issue by NKUShaw** [kylematoba/lcnn#1]
Hi! Can u offer the trained models(pt files) of Standard, LCNNs GradReg and LCNNs + GradReg?
I am trying to reproduce your code. But I found that my result is significantly different with the results in the paper.

**Comment by suraj-srinivas** [comment]
Hi, can you let us know what results you are getting?

**Comment by NKUShaw** [comment]
after training of 200epoches with the command"--model-arch=resnet18lcnn --regularizer=curvature_and_gnorm --dataset=cifar100", I got the low accuracy at 18％

**Comment by kylematoba** [comment]
Hi @NKUShaw, thank you for your interest in our work. I confirm your result on my side. The problem is that you are using the default regularization hyperparameters, which do not agree with what you seem to want. This is described in Appendix E. So: please append `--reg-beta 1e-4 --reg-gamma 1e-5 --reg-gnorm 1e-3` to your call above. 

If you are still having problems, reply here and I can send you trained model weights.
